src/routes/safe/components/Transactions/TxList/TxInfoCreation.tsx

```diff
@@ -96,9 +99,10 @@ export const TxInfoCreation = ({ transaction }: { transaction: Transaction }): R
               {NOT_AVAILABLE}
             </Text>
           )}
-        </div>
+        </StyledGridRow>
       </div>
-      <div className="tx-owners" />
+      <StyledPadding></StyledPadding>
```

Comment:
Dead code.

Reply:
It is a "filler" as the parent is a grid with 2 columns. Let me know if there's a better way do fill it.

Reply:
I'd add a brief comment. You need not include an opening and closing tag as well, e.g. `<StyledPadding />`